docs/detections/alerts-view-details.asciidoc

```diff
@@ -64,8 +64,9 @@ The Insights section provides the following details:
 * *Cases related to the alert* - Shows the total number and names of cases to which the alert has been added. Click a case's name to open its details.
 * *Alerts related by source event* - Shows the ten most recent alerts created by the same source event. This can help you find alerts with a shared origin and provide more context about the source event. Click the *Investigate in timeline* button to examine related alerts in Timeline.
 
-If you have a https://www.elastic.co/pricing[Platinum or Enterprise subscription], these details are also included:
+If you have a https://www.elastic.co/pricing[Platinum or higher subscription], these details are also included:
 
+* *Suppressed alerts* - Indicates that the alert was created with <<alert-suppression,alert suppression>>, and shows how many duplicate alerts were suppressed. This section only appears if alert suppression is enabled for the rule.
```

Comment:
We'll show the suppression insight for any suppressed alerts regardless of license, we only restrict the creation of new suppressed alerts based on the license level.